api/test/test_ts_geojson.py

```diff
@@ -0,0 +1,30 @@
+import json
+
+import datastore_pb2 as dstore
+from deepdiff import DeepDiff
+from formatters.geojson import convert_to_geojson
+from google.protobuf import json_format
+
+
+def actual_response_is_expected_response(actual_response, expected_path, **kwargs):
+    # file_path = Path(Path(__file__).parent, expected_path).resolve()
+    with open(expected_path) as file:
+        expected_json = json.load(file)
+
+    diff = DeepDiff(expected_json, actual_response.json(), **kwargs)
+    assert diff == {}
+
+
+def test_geojson_conversion():
+    ts_response = dstore.GetObsResponse()
+    with open("test/test_data/test_ts_geojson_proto_object.json") as file:
+        json_format.ParseDict(json.load(file), ts_response)
```

Comment:
Can we use `Parse()` like in `create_mock_obs_response`. I think it has slightly different meaning.